src/pages/index.js

```diff
@@ -13,23 +13,30 @@ function HomepageHeader() {
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
         <h1 className={styles.title}>
-          The
-          <br/>
-          Unofficial
-          <br/>
-
-          {siteConfig.title}</h1>
+          The Unofficial
+          <br />
+          {siteConfig.title}
+        </h1>
         <p className={styles.tagline}>{siteConfig.tagline}</p>
         <div className={styles.buttons}>
           <Link
-            className="button button--secondary button--lg"
-            href="https://github.com/Ipmake/kocitylauncher/releases/latest">
+            className={styles.buttonDownload}
+            href="https://github.com/Ipmake/kocitylauncher/releases/latest"
+          >
             Download the Launcher
           </Link>
           <Link
-            className="button button--secondary button--lg"
-            to="/docs/intro">
-            Get Started
+            className={styles.buttonInstructions}
+            href="https://github.com/Ipmake/kocitylauncher#readme"
+          >
+            Instructions and information
```

Comment:
Remove the "Instructions and information" button, 3 buttons are to much for this type of webpage. Maybe add a section below the image of the launcher that has the tutorial on how to install the launcher

Reply:
You're right, 3 is a bit much.

I removed the discord button instead cuz there's a couple other links to discord on the page, and I'd rather have the instructions easy to find.